css/media.css

```diff
@@ -0,0 +1,413 @@
+@media screen and (max-width: 1100px) {
+  .container {
+    max-width: 970px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    max-width: 930px;
+  }
+
+  .header {
+    padding: 27px 0 8px 0;
+  }
+
+  .header__logo {
+    margin-right: 30px;
+    margin-top: 5px;
+  }
+
+  .header__logo-img {
+    width: 61px;
+    height: 55px;
+  }
+
+  .navbar__right-side {
+    margin-top: 3px;
+  }
+
+  .navbar__right-side .navbar__item:first-child {
+    margin-right: 32px;
+  }
+
+  .navbar__right-side .navbar__item:nth-child(2) {
+    margin-right: 28px;
+  }
+
+  .navbar__right-side .navbar__item:nth-child(3) {
+    margin-right: 24px;
+  }
+
+  .navbar__right-side .navbar__item:last-child {
+    margin-right: 0;
+  }
+
+  .search-form {
+    width: 158px;
+    margin-top: 6px; /* for PP */
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654147)